docs/modules/getting_started/pages/first_cluster.adoc

```diff
@@ -0,0 +1,162 @@
+----
+
+Then create the resources by applying the manifest file
+
+[source,bash]
+----
+include::example$code/getting-started.sh[tag=install-zookeeper]
+----
+
+=== HDFS
+
+Druid needs a data storage location, we will use HDFS.
+
+Create `hdfs.yaml` with the following contents:
+
+[source,yaml]
+----
+include::example$code/hdfs.yaml[]
+----
+
+And apply it:
+
+----
+include::example$code/getting-started.sh[tag=install-hdfs]
+----
+
+=== Druid
+
+Create a file named `druid.yaml` with the following contents:
+
+[source,yaml]
+----
+include::example$code/druid.yaml[]
+----
+
+And apply it:
+
+----
+include::example$code/getting-started.sh[tag=install-druid]
+----
+
+This will create the actual druid instance.
+
+== Verify that it works
+
+Next you will submit an ingestion job and then query the ingested data  - either through the web interface or the API.
+
+First, make sure that all the Pods in the StatefulSets are ready:
+
+[source,bash]
+----
+kubectl get statefulset
+----
+
+The output should show all pods ready:
+
+----
+NAME                                 READY   AGE
+simple-druid-broker-default          1/1     5m
+simple-druid-coordinator-default     1/1     5m
+simple-druid-historical-default      1/1     5m
+simple-druid-middlemanager-default   1/1     5m
+simple-druid-router-default          1/1     5m
+simple-hdfs-datanode-default         1/1     6m
+simple-hdfs-journalnode-default      1/1     6m
+simple-hdfs-namenode-default         2/2     6m
+simple-zk-server-default             3/3     7m
+----
+
+Then, port-forward the Druid Router:
+
+----
+include::example$code/getting-started.sh[tag=port-forwarding]
+----
+
+=== Ingest example data
+
+Next, we will ingest some example data using the web interface. If you prefer to use the command line instead, follow the instructions in the collapsed section below.
+
+
+[#ingest-cmd-line]
+.Alternative: Using the command line
+[%collapsible]
+====
+
+If you prefer to not use the user interface and instead interact with the API, create a file `ingestion_spec.json` with the following contents:
+
+[source,json]
+include::example$code/ingestion_spec.json[]
+
+Submit the file with the following `curl` command:
+
+[source,bash]
+include::example$code/getting-started.sh[tag=submit-job]
+
+Continue with the <<_query_the_data,next section>>.
+
+====
+
+To open the user interface navigate your browser to https://localhost:8888/ to find the dashboard:
+
+image::dashboard.png[]
+
+Now load the example data:
+
+image::load_example.png[]
+
+Click through all pages of the load process. You can also follow the https://druid.apache.org/docs/latest/tutorials/index.html#step-4-load-data[Druid Quickstart Guide].
+
+Once you finished the ingestion dialog you should see the ingestion overview with the job, which will eventually show SUCCESS:
+
+image::load_success.png[]
+
+
+
+=== Query the data
+
+Query from the user interface by navigating to the  "Query" interface in the menu and query the `wikipedia` table:
```

Comment:
The first time i saw web interface, now its user interface. Im fine with both, just keep it consistent (ignore the comments above if you go for user interface)